domains/caryak.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Megburns28",
+        "email": "Burn8828@kettering.edu"
+    },
+    "record": {
+        "CNAME": "Megburns28.github.io"
```

Comment:
404 GitHub Page, please provide a preview.

Reply:
I am new to github, im unsure to how to provide a preview

Reply:
Make it so the repository is public so the site is accessible. If you currently visit https://megburns28.github.io you receive a 404 page.